app/src/main/java/uk/org/openseizuredetector/FragmentHrAlg.java

```diff
@@ -2,47 +2,56 @@
 
 import android.content.Context;
 import android.graphics.Color;
+import android.os.Build;
 import android.os.Bundle;
 import android.os.Handler;
 import android.os.Looper;
 import android.util.Log;
 import android.view.LayoutInflater;
 import android.view.View;
 import android.view.ViewGroup;
-import android.widget.Switch;
+import android.widget.CompoundButton;
 import android.widget.TextView;
 
-import androidx.appcompat.app.AppCompatActivity;
+import androidx.annotation.NonNull;
 import androidx.appcompat.widget.SwitchCompat;
 
 import java.util.ArrayList;
 import java.util.Calendar;
+import java.util.Collections;
+import java.util.Iterator;
 import java.util.List;
 import java.util.Objects;
+import java.util.TimeZone;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
 
 import com.github.mikephil.charting.charts.LineChart;
 import com.github.mikephil.charting.data.Entry;
 import com.github.mikephil.charting.data.LineData;
 import com.github.mikephil.charting.data.LineDataSet;
 import com.github.mikephil.charting.utils.ColorTemplate;
+import com.google.android.gms.wearable.Node;
+
 public class FragmentHrAlg extends FragmentOsdBaseClass {
     Context context = getContext();
     Looper looper = null;
-    Handler mHandler =null;
+    Handler mHandler = null;
     TextView tv = null;
     TextView tvCurrent = null;
 
 
     LineChart lineChart;
     LineData lineData;
     LineDataSet lineDataSet;
-    ArrayList hrHistory= new ArrayList<Entry>();
-    ArrayList hrAverages = new ArrayList<Entry>();
-    ArrayList hrHistoryStrings= new ArrayList<Entry>();
-    ArrayList hrAveragesStrings = new ArrayList<Entry>();
+    List<Entry> hrHistory = new ArrayList<>();
+    List<Entry> hrAverages = new ArrayList<>();
+    List<String> hrHistoryStrings = new ArrayList<>();
+    List<String> hrAveragesStrings = new ArrayList<>();
     String TAG = "FragmentOsdAlg";
```

Comment:
We should change this to FragmentHrAlg so we can filter only logcat output from this fragment to make it easier to debug.